Easydict/App/Localizable.xcstrings

```diff
@@ -21,6 +21,22 @@
         }
       }
     },
+    "8080" : {
```

Comment:
Maybe this one should not be classified as a localized string?

Reply:
It seems to be automatically generated by `Text('8080')`.  How do you think it should be changed?

Reply:
Using `String()` to wrap it would work
```swift
Text(String("8080"))
```

Reply:
This works too
```
Text(verbatim: "8080")
```

Reply:
So good!